docs/index.md

```diff
@@ -25,18 +25,28 @@ docker pull jinaai/jina:latest
 ```
 ````
 
-Now that you’re set up, let’s create a project:
+Now that you’re set up, let’s create a project, and start a project to serve:
```

Comment:
I think this part was fine before. Doesn't need a change

Reply:
then propose to remove and I can add it in the batch

Reply:
Done